pkg/controller/machineset/machineset_controller.go

```diff
@@ -230,12 +245,188 @@ func (c *Controller) processNextWorkItem() bool {
 	}
 	defer c.queue.Done(key)
 
+	// create/delete ansible jobs
 	err := c.syncHandler(key.(string))
 	c.handleErr(err, key)
 
+	// create/delete Machine objects
+	err = c.syncHandler2(key.(string))
```

Comment:
We don't want to run two different sync handlers in the same controller. That is going to get very complicated. If we are going to continue to use ansible to create the master machines, then we should have a separate controller for master machine sets.